File: Shared/Responses/Organization/OrganizationResponse.cs
using Shared.Responses.Common;

namespace Shared.Responses.Organization;

public class OrganizationResponse
{
    public string Name { get; set; }
    public string GstNumber { get; set; }
    public AddressResponse Address { get; set; }
    public Guid AdminId { get; set; }
    public Guid Id { get; set; }
    public Guid AddressId { get; set; }
}

File: WebUI/src/app/_responses/Organization-response.ts
import { Guid } from "guid-typescript";
import { AddressResponse } from "./address-response";
import { UserResponse } from "./user-response";

export default class OrganizationResponse {
    id: Guid;
    name: string;
    gstNumber: string;
    address: AddressResponse;
    adminDetails: UserResponse;
    addressId: Guid;
    adminId: Guid;
}